cdap-docs/reference-manual/source/http-restful-api/service.rst

```diff
@@ -69,6 +69,80 @@ The response body will contain a JSON-formatted list of the existing system serv
 See :ref:`downloading System Logs <http-restful-api-logging_downloading_system_logs>` for
 information and an example of using these system services.
 
+Restarting System Service Instances
+------------------------------------
+
+To restart all instances of a system service in CDAP, you can issue an HTTP POST request to the URL::
+
+  POST <base-url>/system/services/<service-id>/restart
+
+You can also restart a particular instance of a system service in CDAP, using the instance id, by issuing
```

Comment:
You can restart a particular instance of a system service in CDAP, using its instance id, by issuing